.github/company.js

```diff
@@ -0,0 +1,128 @@
+    })
+    
+})
+}
+companydata()
+
+let dateLabels = []
+let closePrise = []
+const newUrl = `https://stock-exchange-dot-full-stack-course-
+services.ew.r.appspot.com/api/v3/historical-price-full/${symbol}?serietype=line`
+
+function chart(){
+    
+fetch(newUrl).then(response => {
+    response.json().then(data => {
+        // console.log(data);
+        const historical = (data.historical)
+        for (let i=0; i<historical.length; i++){
+            dateLabels[i] = historical[i].date
+            closePrise[i] = historical[i].close
+        }
+        const chart = {
+            labels: dateLabels,
+            datasets: [{
+              label: 'Stock Price time line',
+              backgroundColor: 'rgb(255, 99, 132)',
+              borderColor: 'rgb(255, 99, 132)',
+              data: closePrise,
+            }]
+          };
+        
+          const config = {
+            type: 'line',
+            data: chart,
+            options: {}
+          };
+          const myChart = new Chart(
+            document.getElementById('myChart'),
+            config
+          );
+    })
+    removeLoader()
+})
+}
+chart()
+
+
+const companyImg = (imgUrl) =>{
+    let img = document.createElement('img');
+    img.src = imgUrl
+    img.classList.add('companyImg')
+    document.querySelector('.titel').appendChild(img)
+}
+
+const companyName = (compantName) =>{
+    let name = document.createElement('h1');
+    name.innerHTML = compantName
+    name.classList.add('companyName')
+    document.querySelector('.titel').appendChild(name)
+}
+
+const companyDescription = (compantDescription) =>{
+    let description = document.createElement('p');
+    description.innerHTML = compantDescription
+    description.classList.add('compantDescription')
+    document.querySelector('.container').appendChild(description)
+}
+const companyWebsite = (comlanyLink) =>{
+    let link = document.createElement('a');
+    let linkText = document.createTextNode(`Company  Website`);
+    link.appendChild(linkText)
+    link.title = (`Company Website`);
+    link.href = (comlanyLink)
+    document.querySelector('.container').appendChild(link)
+    
+}
+const companyStockPrice = (compantStockPrice) =>{
+    let stockPrice = document.createElement('p');
+    stockPrice.innerHTML = 'Stock price ' + compantStockPrice + ' USD'
+    stockPrice.classList.add('compantStockPrice')
+    document.querySelector('.container').appendChild(stockPrice)
+}
+const percentage = (percentage) =>{
+    let changPrecentage = document.createElement('p');
+    changPrecentage.innerHTML ='The percentage change is ' + percentage + " %"
+    if(percentage > 0){
+        changPrecentage.classList.add('greenPrecentage')
+    }
+    else{
+        changPrecentage.classList.add('redPrecentage')
+    }
+    document.querySelector('.container').appendChild(changPrecentage)
+}
+
+function displayLoader() {
+    loader.classList.add('active');
+
+}
+
+function removeLoader() {
```

Comment:
you have to remove the loaded after you got the response from the server, not after a spefic amount of time.